pkg/extensions/search/resolver.go

```diff
@@ -374,13 +281,13 @@ func FilterByTagInfo(tagsInfo []cvemodel.TagInfo) mTypes.FilterFunc {
 	}
 }
 
-func FilterByRepoAndTagInfo(repo string, tagsInfo []cvemodel.TagInfo) mTypes.FilterFunc {
-	return func(repoMeta mTypes.RepoMetadata, manifestMeta mTypes.ManifestMetadata) bool {
+func ProtoFilterByRepoAndTagInfo(repo string, tagsInfo []cvemodel.TagInfo) mTypes.FilterFunc {
+	return func(repoMeta mTypes.RepoMetadata, imageData mTypes.ImageData) bool {
 		if repoMeta.Name != repo {
 			return false
 		}
 
-		manifestDigest := godigest.FromBytes(manifestMeta.ManifestBlob).String()
+		manifestDigest := imageData.Manifests[0].Digest.String()
```

Comment:
Is this correct? Does this work with multiarch?

Reply:
This function is always applied on a manifest level and never on an index level. That's how FilterTags is implemented